api/get_users_handler.go

```diff
@@ -0,0 +1,36 @@
+package api
+
+import (
+	"net/http"
+	"strings"
+
+	"github.com/alphagov/paas-accounts/database"
+	"github.com/labstack/echo"
+)
+
+func GetUsersHandler(db *database.DB) echo.HandlerFunc {
+	return func(c echo.Context) error {
+		type Users struct {
+			Users  []database.User `json:"users"`
+		}
+
+		params := c.QueryParams()
+		users := Users{}
+
+		if len(params["guids"]) > 0 {
+			users.Users, _ = db.GetUserByUUID(strings.Split(params["guids"][0], ","))
+			return c.JSON(http.StatusOK, users)
+		}
+
+		if params.Get("email") != "" {
+			user, err := db.GetUserByEmail(params.Get("email"))
+			if err != nil {
+				return err
+			}
+		    users.Users = append(users.Users, user)
```

Comment:
Formatting

Reply:
I think a straight assignment has a clearer intent over appending to existing results.

Reply:
GetUserByEmail returns a user object so believe it should be appended